### PR TITLE
Adds a round timer for the highlander event

### LINKED ISSUE #8651: Highlander needs a round timer to ensure the round doesn't drag out too long.
### Description

At the moment, highlander ends when the escape shuttle is called and arrives. It should instead have round timer that ends the round automatically after a bit, as highlander is meant to be a fast-paced game mode.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
image

OI FUD, STOAP RUNNING AROOND LIK’ AN EEJIT ‘N’ GIT THAT FUCKIN DISK AFORE TH’ TIME RUNS OOT 

closes #8651 

# Changelog:

CL: [New] - The highlander event now has a timer of 15 minutes so people have a more sense of urgency to get that disk instead of goofing around.

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
This should solve the last issue codacy has, review pls

File: UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		}
+
+		public override void Process(NetMessage msg)
+		{
+			HighlanderTimerUI.Instance.Show(msg.Time);
+		}
+
+		public static void Send(ConnectedPlayer player)
+		{
+			var highlanderEvent =  InGameEventsManager.Instance.ListOfAntagonistEventScripts.Find(x => x.GetType() == typeof(EventHighlander))
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs:22]: what is this?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs:22]: It gets the event so it grabs the time thats on it and send it to the player

Comment (corp-0) [reply UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs:22]: how often? It also has a GetComponent

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs:22]: only once, it doesn't get spammed.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Messages/Server/HighlanderTimerMessage.cs:22]: hewwwooo?

**Review by corp-0 — APPROVED** [review]

**Pull request merged**